tests/RobotFramework/tests/cumulocity/software_management/software-update-child.robot

```diff
@@ -8,6 +8,12 @@ Test Setup       Custom Setup
 Test Teardown    Custom Teardown
 
 *** Test Cases ***
+Supported software types should be declared during startup
+    [Documentation]    #2654 This test will be updated once advanced software management support is implemented
+    ...    FIXME: Why this one fails?
```

Comment:
In my dev machine, this test always fails although I could confirm that the capability messages are sent by entering a test container. Please help me here. The same test case for parent device works.

Reply:
I can confirm that the expected messages are actually published.

This seems to be a missing `Set Suite Variable    $DEVICE_SN`.

Reply:
> This seems to be a missing Set Suite Variable    $DEVICE_SN.

This one exists in Custom Setup, which runs in test level...